src/main/java/org/folio/innreach/controller/d2ir/InnReachCirculationController.java

```diff
@@ -30,9 +34,27 @@
 public class InnReachCirculationController implements InnReachCirculationApi {
 
   private final CirculationService circulationService;
+  private final RequestService requestService;
 
   @Override
-  @PostMapping("/patronhold/{trackingId}/{centralCode}")
+  @PostMapping("/itemhold/{trackingId}/{centralCode}")
+  public ResponseEntity<InnReachResponseDTO> createInnReachTransactionItemHold(@PathVariable String trackingId,
+                                                                               @PathVariable String centralCode,
+                                                                               TransactionHoldDTO dto) {
+    var response = circulationService.createInnReachTransactionItemHold(trackingId, centralCode, dto);
+    HttpStatus status;
+    if (response.getStatus().equals("ok")) {
+      status = HttpStatus.OK;
+      requestService.createItemHoldRequest(trackingId);
+    } else {
+      status = HttpStatus.BAD_REQUEST;
+    }
```

Comment:
no need to check for response status here, it will always return `success`, since the error response will be constructed by `D2irExceptionHandlerController`